app/src/androidTest/java/com/github/fribourgsdp/radio/UserProfileActivityTest.kt

```diff
@@ -2,29 +2,113 @@ package com.github.fribourgsdp.radio
 
 import android.content.Context
 import android.content.Intent
-import androidx.test.core.app.ActivityScenario
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.espresso.Espresso
 import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.intent.Intents
 import androidx.test.espresso.intent.matcher.IntentMatchers
+import junit.framework.TestCase
 import androidx.test.espresso.matcher.ViewMatchers
 import androidx.test.ext.junit.rules.ActivityScenarioRule
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.android.gms.tasks.Tasks
 import com.google.firebase.auth.FirebaseAuth
-import junit.framework.TestCase
 import org.hamcrest.Matchers
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+
+import androidx.test.espresso.matcher.ViewMatchers.*
+
+
 import java.util.concurrent.TimeUnit
 
 @RunWith(AndroidJUnit4::class)
 class UserProfileActivityTest : TestCase() {
     @get:Rule
     var userProfileActivityRule = ActivityScenarioRule(UserProfileActivity::class.java)
 
+
+   // @get:Rule
+    //var userProfileActivityRule = ActivityScenarioRule(UserProfileActivity::class.java)
+
+    private val ctx: Context = ApplicationProvider.getApplicationContext()
+
+    /*@Test
+    fun userSavedLocallyIsDisplayed(){
+        Intents.init()
+
+        val testUser = User("TestSavedLocally")
+        testUser.save(ctx)
+        val intent: Intent = Intent(ctx, UserProfileActivity::class.java)
+
+        ActivityScenario.launch<UserProfileActivity>(intent).use { scenario ->
+
+            onView(withId(R.id.username)).
+            check(matches(withText(testUser.name)));
+
+
+        }
+        Intents.release()
+
+    }*/
```

Comment:
If you don't plan on using this test, you should delete this. If not, you should add a comment explaining why you commented all of this.

Reply:
Same as below